app/models/papi_client.rb

```diff
@@ -321,8 +322,11 @@ def get_command_line(study_file:, action:, user_metrics_uuid:, params_object: ni
     when 'ingest_cluster'
       command_line += " --cluster-file #{study_file.gs_url} --ingest-cluster"
     when 'ingest_anndata'
-      # extract cluster data from AnnData file, currently hardcoding the obsm-keys
-      command_line +=  " --ingest-anndata --anndata-file #{study_file.gs_url} --extract-cluster --obsm-keys ['X_umap','X_tsne']"
+        # extract cluster data from AnnData file, currently hardcoding the obsm-keys
+        command_line +=  " --ingest-anndata --anndata-file #{study_file.gs_url} --extract-cluster --obsm-keys ['X_umap','X_tsne']"
+    when 'ingest_anndata_reference'
+        command_line = "python ingest_pipeline.py --study-id #{study.id} --study-file-id #{study_file.id} " \
```

Comment:
We've already initializes `command_line` with the default values so no need to re-declare them here:
```suggestion
        command_line +=  " --ingest-anndata --anndata-file #{study_file.gs_url}"
```

Reply:
The initialized `command_line` includes the action which in this case we don't want to be part of the command to ingest pipeline since the action`ingest_anndata_reference` isn't part of the ingest command. I made an update to duplicate less of the command_line, let me know what you think.

Reply:
Ah yes excellent point!  I missed that.